guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -33,7 +33,7 @@ If these files have been deleted, they must be restored from a backup in order f
 ====
 endif::[]
 
-.Upgrading {SmartProxyServer}s
+.Upgrading {SmartProxyServers}
```

Comment:
It's probably not stated anywhere, but from what I can tell we also keep unnumbered titles camel-cased?
```suggestion
.Upgrading {SmartProxyServersTitle}
```

Reply:
Yes, we should do this. I will open a new PR for that against master and then start cherry-picking. This PR is already a cherry-pick from master.

Reply:
Sounds good to me!